src/screens/game_over.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+Lootr.Screens.GameOver = {
+    _caption: "Game Over Screen",
+
+    enter: function( gameDisplay ) {
+        console.log("Entered Game Over Screen");
+
+        this._gameDisplay = gameDisplay;
+    },
+
+    renderGame: function () {
+		let player = Lootr.getPlayer();
+		console.log('has health', player.hasComponent('Health'));
+
+		this._gameDisplay.drawText(1, 1, "Game Over Screen");
+    },
+
+    exit: function() {
+        console.log("Exited Game Over Screen");
+    },
+
+    handleInput( inputType, inputData ) {
+		Logger("Resurrecting!!")
+		
+		let player = Lootr.getPlayer();
+
+		let hp = player.getComponent("Health").hp = 10;
```

Comment:
I think it is normally 100, but we should handle starting health in a more sophisticated way eventually anyway.